docs/en/docs/advanced/custom-response.md

```diff
@@ -202,6 +202,10 @@ Takes an async generator or a normal generator/iterator and streams the response
 {!../../../docs_src/custom_response/tutorial007.py!}
 ```
 
+For reference, `StreamingResponse` comes from [`starlette`](https://github.com/encode/starlette).
+More information on it can be found in Starlette's docs:
+https://www.starlette.io/responses/#streamingresponse.
```

Comment:
```suggestion
You can read more details about the <a href="https://www.starlette.io/responses/#streamingresponse" class="external-link" target="_blank">`StreamingResponse` object in the official Starlette documentation site</a>.
```

Reply:
Thank you for the suggestion, but I prefer to stick with Markdown built-in links, as opposed to new-tab opening. Most of the links in this doc do not use `target="_blank"` method. If others want a new tab, they can CTRL-click as is standard across the internet.